src/Explorer/Controls/NotebookGallery/Cards/GalleryCardComponent.tsx

```diff
@@ -103,7 +103,7 @@ export class GalleryCardComponent extends React.Component<GalleryCardComponentPr
             </Card.Item>
 
             <Card.Section styles={{ root: { padding: GalleryCardComponent.cardItemGapBig } }}>
-              <Text variant="small" nowrap>
+              <Text variant="small" nowrap styles={{ root: { height: 18 } }}>
```

Comment:
why inlined this height when you have constants at the beginning defining styles?